openedx/features/partners/tests/test_reset_password.py

```diff
@@ -0,0 +1,59 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from openedx.features.partners.tests.factories import PartnerUserFactory, PartnerFactory
+from student.tests.factories import UserFactory
+
+
+class ResetPasswordTestCases(APITestCase):
+
+    def setUp(self):
+        self.partner_reset_password_end_point = reverse('partner_reset_password')
+        self.user = UserFactory(username='testuser', password='12345', email='abc@test.com')
```

Comment:
I don't think we need to pass these arguments to create a user from `UserFactory`. Please see if we can just create and use one using `UserFactory()` initializer.